.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+INSTRUCTIONS
+==========
+
+If you do not follow the below instructions, your issue may be dismissed.
+
+1. Search existing issues to avoid creating duplicates.
+
+2. If possible test using the latest release to make sure your issues has not already been fixed: 
+https://github.com/Azure/azure-iot-sdk-c/releases/latest 
+
+3. Do not share information from your Azure subscription here (connection strings, service names (IoTHub, Provisioning), etc...). Do so through your Azure subscription.
```

Comment:
I'd clarify service names, maybe reword that to "IoT Hub name" and "Device Provisioning Service scope ID".

And instead of "do so through your Azure subscription", please use this "Please submit a ticket and get assistance from Microsoft support team using the following link:
https://docs.microsoft.com/en-us/azure/azure-supportability/how-to-create-azure-support-request"